azure-functions-codegen/Cargo.toml

```diff
@@ -15,5 +15,11 @@ proc-macro = true
 azure-functions-shared = { version = "0.3.0", path = "../azure-functions-shared" }
 quote = "0.6.11"
 syn = { version = "0.15.26", features = ["full"] }
-proc-macro2 = { version = "0.4.26", features = ["nightly"] }
+proc-macro2 = { version = "0.4.26" }
 lazy_static = "1.2.0"
+proc-macro-hack = "0.5.4"
+cfg-if = "0.1.6"
+
+
+[features]
+unstable = []
```

Comment:
I opted for the name `unstable` since this is what serde uses. Unfortunately there's no really good way to automatically set this feature when a user opts for the nightly toolchain. I think we just need to document this.

Reply:
👍